File: tutorial1/NebulaTutorial1/TodoEditViewController.m
//
//  TodoEditViewController.m
//  NebulaTutorial1
//  Copyright (c) 2014-2018, NEC Corporation. All rights reserved.
//

#import "TodoEditViewController.h"
#import "TodoListViewController.h"

@implementation TodoEditViewController
{
    __weak IBOutlet UITextView *textView;
}

- (void)viewDidLoad
{
    [super viewDidLoad];

    textView.text = self.todoText;

    [textView becomeFirstResponder];
}

- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
{
    self.todoText = textView.text;
}

@end

File: tutorial1-swift/NebulaTutorial1/TodoEditViewController.swift
//
//  TodoEditViewController.swift
//  Copyright (c) 2014-2018, NEC Corporation. All rights reserved.
//

import UIKit

class TodoEditViewController : UIViewController {
    var todoText : String = ""
    
    @IBOutlet weak var textView : UITextView?
    
    override func viewDidLoad() {
        super.viewDidLoad()
    
        self.textView!.text = self.todoText;
        self.textView!.becomeFirstResponder()
    }
    
    override func prepare(for segue: UIStoryboardSegue, sender: Any!) {
        self.todoText = self.textView!.text;
    }
}
